addons/spreadsheet/static/src/pivot/pivot_data_source.js

```diff
@@ -61,19 +61,28 @@ export class PivotDataSource extends OdooViewsDataSource {
     }
 
     /**
-     * @param {string[]} domain
+     * High level method computing the result of ODOO.PIVOT.HEADER functions.
+     * - regular function 'ODOO.PIVOT.HEADER(1,"stage_id",2,"user_id",6)'
+     * - measure header 'ODOO.PIVOT.HEADER(1,"stage_id",2,"user_id",6,"measure","expected_revenue")
+     * - positional header 'ODOO.PIVOT.HEADER(1,"#stage_id",1,"#user_id",1)'
+     *
+     * @param {(string | number)[]} functionArgs arguments of the function (except the first one which is the pivot id)
+     * @returns {string | number}
      */
-    getDisplayedPivotHeaderValue(domain) {
+    computeOdooPivotHeaderValue(functionArgs) {
```

Comment:
or `pivotArgs` ? I agree `functionArgs` is rubbish

Reply:
`pivotFormulaArgs` ? Idk. I still think `domain` was somewhat better. I'd mention either `domain` or `formula` (because function is technically more correct, but is confusing when we're reading... actual JS functions)